sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/HiveSessionImpl.java

```diff
@@ -223,6 +223,18 @@ private void configureSession(Map<String, String> sessionConfMap) throws HiveSQL
 
   @Override
   public void setOperationLogSessionDir(File operationLogRootDir) {
+    if (!operationLogRootDir.exists()) {
+      LOG.warn("The operation log root directory is removed, recreating:" +
```

Comment:
nit: extra space at the end of line

Reply:
sorry @jiangxb1987 I can't see any extra space at the end of this line...

Reply:
oh sorry, now I understand, thanks! Yes, in Hive this space is still missing, I see only now. Thanks.